subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -243,21 +226,10 @@ void ull_sched_mfy_win_offset_use(void *param)
 	}
 
 	/*
-	 * TODO: update when updating the connection update procedure
+	 * TODO: update calculationg of the win_offset
+	 * when updating the connection update procedure
+	 * see the legacy code from Zephyr v3.2 for inspiration
```

Comment:
Should this refer to v3.3?

Reply:
correct. 3.3 is the latest release with legacy in it